### PR TITLE
feat(cli): `--quiet` does not print template in `cdk synth`

### LINKED ISSUE #11970: (cli): cdk synth need to disable printing the Cloudformation Template to console
### The Question
There should be a way to run cdk synth and disable printing the Cloudformation Template to stdout. 
It would make sense for this to work when use --output or for it to be its own flag. 

The reason: In our Codebuild logs it is very annoying to have a giant template printed line by line when we are only running the synth to use the --output flag to analyse the Cloudformation Template. 

### Environment

  - **CDK CLI Version:** all
  - **Module Version:** all
  - **Node.js Version:** n/a
  - **OS:** all
  - **Language (Version):** TypeScript / possibly all

### PR DESCRIPTION
**Pull request by briete**
fixes #11970 

---
_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license_

### CODE REVIEW

File: packages/aws-cdk/bin/cdk.ts [hunk 1]
```diff
@@ -68,7 +68,8 @@ async function parseCommandLineArguments() {
       .option('long', { type: 'boolean', default: false, alias: 'l', desc: 'Display environment information for each stack' }),
     )
     .command(['synthesize [STACKS..]', 'synth [STACKS..]'], 'Synthesizes and prints the CloudFormation template for this stack', yargs => yargs
-      .option('exclusively', { type: 'boolean', alias: 'e', desc: 'Only synthesize requested stacks, don\'t include dependencies' }))
+      .option('exclusively', { type: 'boolean', alias: 'e', desc: 'Only synthesize requested stacks, don\'t include dependencies' })
+      .option('no-stdout', { type: 'boolean', desc: 'Do not output CloudFormation Template to stdout', default: false }))
```
Comment (rix0rrr) [inline packages/aws-cdk/bin/cdk.ts:72]: Please call this `quiet` and alias to `q`.

**Review by rix0rrr — CHANGES_REQUESTED** [review]

**Comment by briete** [comment]
@rix0rrr Please review again🙇

File: packages/aws-cdk/README.md [hunk 2]
```diff
@@ -115,6 +115,9 @@ $ cdk synth
 
 $ # Synthesize cloud assembly for StackName, but don't include dependencies
 $ cdk synth MyStackName --exclusively
+
+$ # Synthesize cloud assembly for StackName, but don't cloudFormation template output to STDOUT
+$ cdk synth MyStackName --no-stdout=true
```
Comment (relm923) [inline packages/aws-cdk/README.md:120]:
```suggestion
$ cdk synth MyStackName --quiet
```
?

Comment (briete) [reply packages/aws-cdk/README.md:120]: sorry.

**Review by rix0rrr — APPROVED** [review]

File: packages/aws-cdk/README.md [hunk 3]
```diff
@@ -115,6 +115,9 @@ $ cdk synth
 
 $ # Synthesize cloud assembly for StackName, but don't include dependencies
 $ cdk synth MyStackName --exclusively
+
+$ # Synthesize cloud assembly for StackName, but don't cloudFormation template output to STDOUT
```
Comment (pgollucci) [inline packages/aws-cdk/README.md:119]: .... 'output' cloud formation .......

**Pull request merged**